apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -747,8 +749,30 @@
             }
           })
 
+
+
+
           if (this.userKeymap === "vim") {
             const codeMirrorVimInstance = document.querySelector(".CodeMirror").CodeMirror.constructor.Vim
+            codeMirrorVimInstance.defineEx("write", "w", this.triggerSave)
+            codeMirrorVimInstance.defineEx("quit", "q", this.close)
+            codeMirrorVimInstance.defineEx("qa", "qa", () => {this.$root.$emit(AppEvent.closeAllTabs)})
+            codeMirrorVimInstance.defineEx("x", "x", () => {
+              this.triggerSave()
+              if(this.query.id) {
+                this.close()
+              }
+            })
```

Comment:
Is it easy for you to add common aliases for commands like this? I know I use `:wq` often instead of `:x`